<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGirisEkrani</class>
 <widget class="QWidget" name="frmGirisEkrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kullanıcı Girişi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frmGirisEkrani{
	 background-color:#004b66;
}</string>
  </property>
  <widget class="QPushButton" name="btnGiris">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnGiris {
    background-color:#ff8521; /* Kırmızımsı bir ton */
    color:#004b66;           /* Beyaz yazı rengi */
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    border-radius: 20px;
    padding: 10px;
    border: 2px solid #ff8521;
}

#btnGiris:hover {
    background-color:#f4c997; /* Hover durumu için daha koyu ton */
    color: #004b66;
}

#btnGiris:pressed {
    background-color: #ff8521; /* Basıldığında daha koyu kırmızı */
    border: 2px solid #FFFFFF;
}
</string>
   </property>
   <property name="text">
    <string>Giriş Yap </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lbllKullaniciAdi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:#f4c997; /* Kırmızımsı bir ton */
    color:#004b66;           /* Beyaz yazı rengi */
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    border-radius: 25px;
    padding: 10px;
    border: 2px solid #f4c997;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Kullanıcı adı</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lblSifre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:#f4c997; /* Kırmızımsı bir ton */
    color:#004b66;           /* Beyaz yazı rengi */
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    border-radius: 25px;
    padding: 10px;
    border: 2px solid #f4c997;</string>
   </property>
   <property name="placeholderText">
    <string>Şifre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCikis">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnCikis {
    background-color:#ff8521; /* Kırmızımsı bir ton */
    color:#004b66;           /* Beyaz yazı rengi */
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    border-radius: 20px;
    padding: 10px;
    border: 2px solid #ff8521;
}

#btnCikis:hover {
    background-color:#f4c997; /* Hover durumu için daha koyu ton */
    color: #004b66;
}

#btnCikis:pressed {
    background-color: #ff8521; /* Basıldığında daha koyu kırmızı */
    border: 2px solid #FFFFFF;
}
</string>
   </property>
   <property name="text">
    <string>Çıkış Yap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: #FFD700; /* Sarı renk */
    font-size: 48px; /* Yazı boyutu */
    font-weight: bold; /* Kalın yazı */
    text-transform: uppercase; /* Büyük harf yapma */
</string>
   </property>
   <property name="text">
    <string>Kullanıcı</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: white; /* Sarı renk */
    font-size: 48px; /* Yazı boyutu */
    font-weight: bold; /* Kalın yazı */
    text-transform: uppercase; /* Büyük harf yapma */
</string>
   </property>
   <property name="text">
    <string>Girişi</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Sistem Resimleri/arka_planlar/arkaplanlar.qrc"/>
  <include location="../Sistem Resimleri/butonlar/butonlar.qrc"/>
 </resources>
 <connections/>
</ui>
